Portal_Application/Controllers/ArtistController.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Core;
+using Interfaces.Models;
+
+namespace Portal_Application.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ArtistController : ControllerBase
+    {
+        private readonly ILogger<ArtistController> _logger;
+        private readonly ApplicationContext _db;
```

Comment:
Using DBContext in the controller it is an antipattern, I don't think you need it as part of the current PR, but if you want to leave it you need to move to the separate file repository pattern implementation to the Infrastructure layer and interface of the Repository in the Domain/Interfaces.